README.md

```diff
@@ -28,7 +39,7 @@ julia> using AutoAligns
 
 julia> aa = AutoAlign(align = Dict(1 => left, :default => right));
 
-julia> for (i,r) in zip([1,100,10000],["a","bb","ccc"])
+julia> for (i, r) in zip([1, 100, 10000], "a", "bb", "ccc"])
            print(aa, r)
```

Comment:
I think you removed an `[` that belongs there (before the `"a"`), can you put it back?

Reply:
Oops! Sorry about that :)